example/src/new_api/camera/index.tsx

```diff
@@ -20,7 +20,10 @@ export default function Home() {
   const [selectedFilter, setSelectedFilter] = useState(0);
   const [isRecording, setIsRecording] = useState(false);
   const [remainingTimeMs, setRemainingTimeMs] = useState(MAX_VIDEO_DURATION_MS);
-  const [recordingIntervalHandle, setRecordingIntervalHandle] = useState(-1);
+  const [
+    recordingIntervalHandle,
+    setRecordingIntervalHandle,
+  ] = useState<NodeJS.Timeout | null>(null);
```

Comment:
IIRC `setTimeout` return value was different on the web and RN. Use `ReturnType<typeof setTimeout>` instead.

Reply:
Done in https://github.com/software-mansion/react-native-gesture-handler/pull/1882/commits/08e2ff1a328b5da72dcf726cd2f236127c3fdef1.